abm.py

```diff
@@ -22,7 +22,7 @@
 History = Iterator[Tuple[List[Dict[str, Any]], Dict[str, Any]]]
 
 
-class UnorderedSpace:
+class GeographicalRegion:
```

Comment:
I don't think this name makes sense, because the agents are all in one giant bag - there's nothing to do with regions.

Reply:
I am happy with all the other changes but still not sure what to do about this one.
I get that for us it is a `GeographicalRegion`, but it is more general than that. I think it could be named to `DiscreteSpace`, since each item in that space is discrete. I think everything in the `abm.py` should be named quite generally. 

I'm happy to set `GeographicalRegion = DiscreteSpace` in our `simulation/...` though and use it there in the `DomesticHeatingABM` - WDYT?

Reply:
thanks both - I'll rename it back to `UnorderedSpace` for now.

Reply:
Sure, for convenience you could set `GeographicalRegions = UnorderedSpace` in `simulation/model.py` just above the `DomesticHeatingABM` and then change [this line](https://github.com/centrefornetzero/domestic-heating-abm/blob/76a372c9828b3a1ba43b4a5a4f1dca7c3d080268/simulation/model.py#L38) to:
```
super().__init__(space=GeographicalRegions)
```
if that helps it make more sense...
But will leave that up to you.